add IOUtil.unrollFiles(INPUT, IOUtil.VCF_EXTENSIONS); to MergeVcfs

lindenb opened the pull request:
just like in GatherVcfs I've added 

```
INPUT = IOUtil.unrollFiles(INPUT, IOUtil.VCF_EXTENSIONS);
for (final File f: INPUT) IOUtil.assertFileIsReadable(f);
```

in MergeVcfs to support the 'list' extension

### Checklist

#### Content
- [ ] Added or modified tests to cover changes and any new functionality
- [ ] Edited the README / documentation (if applicable)
- [x ] All tests passing on Travis

#### Review
- [ ] Final thumbs-up from reviewer
- [ ] Rebase, squash and reword as applicable

For more detailed guidelines, see https://github.com/broadinstitute/picard/wiki/Guidelines-for-pull-requests

yfarjoun commented on src/main/java/picard/vcf/MergeVcfs.java:
```diff
@@ -94,6 +94,8 @@ public MergeVcfs() {
     protected int doWork() {
         final ProgressLogger progress = new ProgressLogger(log, 10000);
         final List<String> sampleList = new ArrayList<String>();
+        INPUT = IOUtil.unrollFiles(INPUT, IOUtil.VCF_EXTENSIONS);
```
please update documentation to inform users of this option.

tfenne replied:
Came here to say this.  Could you just add a mention of the capability to the doc for the option?

yfarjoun commented on src/main/java/picard/vcf/MergeVcfs.java:
```diff
@@ -94,6 +94,8 @@ public MergeVcfs() {
         final ProgressLogger progress = new ProgressLogger(log, 10000);
         final List<String> sampleList = new ArrayList<String>();
+        INPUT = IOUtil.unrollFiles(INPUT, IOUtil.VCF_EXTENSIONS);
+        for (final File f: INPUT) IOUtil.assertFileIsReadable(f);
```
```
INPUT.stream.foreach(IOUtil::assertFileIsReadable)
```

lindenb commented:
I've updated the doc for INPUT, and I've removed the line 
```
for(final File f: INPUT) IOUtil.assertFileIsReadable(f); 
```
because anyway, this test is called later in the source.

tfenne commented:
👍 from me.  @yfarjoun any reason not to merge?

yfarjoun commented:
I didn't look. but I trust you did before giving it the thumbs up...:smile:

lindenb commented:
#gentlereminder anyone for this ?
